activerecord/test/cases/associations/callbacks_test.rb

```diff
@@ -7,7 +7,7 @@
 require 'models/company'
 
 class AssociationCallbacksTest < ActiveRecord::TestCase
-  fixtures :posts, :authors, :projects, :developers
+  fixtures :posts, :authors, :author_addresses, :projects, :developers
```

Comment:
Yes.

We must use `authors` fixture with `author_addresses` because of its foreign key constraint.